### ISSUE TITLE
Install of Forge 1.17.1-37.0.10 fails, forgeJson.path is null

### ISSUE THREAD

**Issue by MCOfficer** [gorilla-devs/GDLauncher#1004]
**Describe the bug**
Installing Forge v1.17.1-37.0.10 fails with the game being stuck at "100% Downloading game files" or "100% Downloading Forge" (presumably if the forge installer isn't cached).

**To Reproduce**
1. Open GDLauncher v1.1.11
2. Open the dev console
3. Click on + to install a new instance
4. Choose Forge > 1.17.1 > 37.0.10
5. proceed with install
6. watch the dev console for an error, in my case 

```
index.js:39 Uncaught (in promise) TypeError: Cannot read property 'split' of null
    at D (index.js:39)
    at actions.js:1293
    at l (runtime.js:63)
    at Generator._invoke (runtime.js:293)
    at Generator.next (runtime.js:118)
    at r (asyncToGenerator.js:3)
    at s (asyncToGenerator.js:25)
```

**Expected behavior**
Installation should work, or at least give an error instead of stalling.

**Possible solutions**
I don't know about solutions, but i can tell you the cause. If you fetch yourself the `1.17.1-37.0.10.jar` (or get it from datastore/forgeInstallers) and check the `install_profile.json` inside, you'll notice the `path` is null, which causes the exception above.

I'll also attach it below.

**Operating System:**
Should be OS-agnostic.
 - OS: Windows
 - Version 10

**Additional context**

[37.0.10_install_profile.json.zip](https://github.com/gorilla-devs/GDLauncher/files/6878827/37.0.10_install_profile.json.zip)

**Comment by MCOfficer** [comment]
Also reproduced this with 37.0.9 and 37.0.0. Seems to me like this is behind other forge 1.17.1 failures, such as #998

**Comment by hypherionmc** [comment]
Forge 1.17.1 is broken on almost all launchers. This release of forge is a "beta" and they aren't planning to get it stable, because they are aiming for 1.18

**Comment by MCOfficer** [comment]
> 
> 
> Forge 1.17.1 is broken on almost all launchers. This release of forge is a "beta" and they aren't planning to get it stable, because they are aiming for 1.18

It still feels weird - I will admit to not being familiar with forge internals, but that seems like a packaging change, nothing to do with ingame stability.

**Comment by hypherionmc** [comment]
A couple of nights ago I messed with the code to get it to install, which it did, but in the process, the actual forge installer errored out. Not the launcher, the installer itself

**Comment by ramonGonzEdu** [comment]
Shouldn't GDLauncher sanitize the JSON it reads rather than blindly trust it? Like checking to see if the properties actually exist and making a popup message if they don't?

**Comment by Shad0wyDr3ams** [comment]
Copied my comment from #1008

this should be fixable via adding this to path line
"path": "net.minecraftforge:forge:1.17.1-37.0.12"

the reason i say this is because in every other version of forge before 1.17-1.17.1, that is how the path is named with the exception of the numbers being different per version.

To my knowledge GDLauncher has no way of allowing custom forge installs... this being said forge itself needs to implement the fix or gd launcher needs to implement a bypass in their launcher so it doesnt read the path line. Or give people the option to do custom forge installs.

**Comment by hypherionmc** [comment]
Fixed in release 1.1.12+
